src/openforms/templates/master.html

```diff
@@ -29,6 +29,14 @@
 </head>
 
 <body class="view__body">
+    {% if enable_analytics and analytics_tools_config.enable_google_analytics and analytics_tools_config.gtm_code %}
+        {# Load the second part of the Google tag manager snippet #}
+        <!-- Google Tag Manager (noscript) -->
+        <noscript><iframe class="google-tag-manager__noscript" src="https://www.googletagmanager.com/ns.html?id={{ analytics_tools_config.gtm_code }}"
```

Comment:
this is not correct BEM naming - an element cannot exist without a parent block (there's no `.google-tag-manager` class anywhere)